### PR TITLE
docs(common): Updated wax boilerplate readme #441

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?
Updated wax boilerplate readme and added About Edenia section.

- Resolves https://github.com/eoscostarica/guide.eoscostarica.io/issues/441

### Steps to test

1. Open readme and see new section and grammar changes

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,41 +142,21 @@ We need to handle REST custom requests coming from the Hasura GraphQL server. Fo
...
-      <img src="https://avatars3.githubusercontent.com/u/10634375?s=460&v=4" width="100px;" alt="Teto Gomez"/><br /><sub><b>Teto Gomez</b></sub></a><br /><a href="https://github.com/eoscostarica/eosrate/commits?author=tetogomez" title="Ideas, Planning, & Feedback">🤔</a> <a href="https://github.com/eoscostarica/eosrate/commits?author=tetogomez" title="Code">💻</a> <a href="#review-tetogomez" title="Reviewed Pull Requests">👀</a></td>
-      <td align="center"><a href="https://github.com/adriexnet">
-      <img src="https://avatars3.githubusercontent.com/u/5375168?s=460&u=542a27a00b761d98851991c6a6d5f78d7b35a2b2&v=4" width="100px;" alt="Adriel Diaz"/><br /><sub><b>Adriel Diaz</b></sub></a><br /><a href="https://github.com/eoscostarica/eosrate/commits?author=adriexnet" title="Ideas, Planning, & Feedback">🤔</a> <a href="https://github.com/eoscostarica/eosrate/commits?author=adriexnet" title="Code">💻</a> <a href="#review-adriexnet" title="Reviewed Pull Requests">👀</a></td>
-  </tr>
-</table>
-
-This project follows the [all-contributors](https://github.com/kentcdodds/all-contributors) specification. Contributions of any kind welcome!
-
-## About EOS Costa Rica
```
Comment (xavier506) [inline README.md:178]: Add About Edenia Header

**Review by xavier506 — CHANGES_REQUESTED** [review]
missing header and table of contents link for about edenia

File: README.md (hunk 1 above)
Comment (fvives99) [reply README.md:178]: The suggested change is already on the commit

**Review by fvives99 — COMMENTED** [review]
The suggested change is already on the commit

**Pull request merged**